packages/data/src/reports/actions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Internal dependencies
+ */
+import { getResourceName } from '../utils';
+import TYPES from './action-types';
+
+export function setItemError( endpoint, query, error ) {
+	const resourceName = getResourceName( endpoint, query );
+
+	return {
+		type: TYPES.SET_ITEM_ERROR,
+		resourceName,
+		error,
+	};
+}
+
+export function setReportItems( endpoint, query, items ) {
+	const resourceName = getResourceName( endpoint, query );
+
+	return {
+		type: TYPES.SET_REPORT_ITEMS,
+		resourceName,
+		items,
+	};
+}
+
+export function setReportStats( endpoint, query, stats ) {
+	const resourceName = getResourceName( endpoint, query );
+
+	return {
+		type: TYPES.SET_REPORT_STATS,
+		resourceName,
+		stats,
+	};
+}
+
+export function setStatError( endpoint, query, error ) {
+	const resourceName = getResourceName( endpoint, query );
```

Comment:
Oh wow, that's a great catch.  It seems like this would have worked incorrectly before as well; maybe we were lucky not to hit it?  I'll get this fixed up.

Reply:
On checking into this, it looks like the values are included.  Are you seeing something different?  Example of the identifier on the resource name:

<img width="530" alt="Screen Shot 2020-08-13 at 9 13 13 AM" src="https://user-images.githubusercontent.com/10561050/90139400-fd184500-dd80-11ea-92fb-1fa09c9206f1.png">

Reply:
Oh duh, reading is fundamental!. The `identifier` object is passed in its entirety